code/modules/fish/fish_tank.dm

```diff
@@ -402,36 +402,38 @@
 
 	examine_message += "Water level: "
 
-	if(water_level == 0)
-		examine_message += "\The [src] is empty! "
-	else if(water_level < water_capacity * 0.1)
-		examine_message += "\The [src] is nearly empty! "
-	else if(water_level <= water_capacity * 0.25)
-		examine_message += "\The [src] is about one-quarter filled. "
-	else if(water_level <= water_capacity * 0.5)
-		examine_message += "\The [src] is about half filled. "
-	else if(water_level <= water_capacity * 0.75)
-		examine_message += "\The [src] is about three-quarters filled. "
-	else if(water_level < water_capacity)
-		examine_message += "\The [src] is nearly full! "
-	else if(water_level == water_capacity)
-		examine_message += "\The [src] is full! "
+	switch (water_level/water_capacity)
+		if (0)
+			examine_message += "\The [src] is empty! "
+		if (0.001 to 0.1)
```

Comment:
`0.001`
I know what you're thinking, but... it works, up to 0.25u water, which is, I think, the smallest fraction you can get.